src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -101,7 +101,10 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
     register_cli_argument(scope + ' deployment source', 'repo_url', options_list=('--repo-url', '-u'), help='repository url to pull the latest source from, e.g. https://github.com/foo/foo-web')
     register_cli_argument(scope + ' deployment source', 'branch', help='the branch name of the repository')
     register_cli_argument(scope + ' deployment source', 'cd_provider', help='type of CI/CD provider', default='kudu', **enum_choice_list(['kudu', 'vsts']))
-    register_cli_argument(scope + ' deployment source', 'cd_app_type', arg_group='VSTS CD Provider', help='web application framework you used to develop your app', default='AspNetWap', **enum_choice_list(['AspNetWap', 'AspNetCore', 'NodeJSWithGulp', 'NodeJSWithGrunt']))
+    register_cli_argument(scope + ' deployment source', 'cd_app_type', arg_group='VSTS CD Provider', help='web application framework you used to develop your app', default='AspNetWap', **enum_choice_list(['AspNetWap', 'AspNetCore', 'NodeJS', 'PHP', 'Python']))
+    register_cli_argument(scope + ' deployment source', 'nodejs_task_runner', arg_group='VSTS CD Provider', help='Task runner for nodejs. Default : None', default='None', **enum_choice_list(['None', 'Gulp', 'Grunt']))
```

Comment:
Which tasks is the build pipeline composed of when the `None` task runner is selected? Simply running `npm install`? Is there any support (planned?) for having it create an NPM scripts based pipeline (e.g. it adds a task that runs `npm run build`)?